assets/common/js/game.js

```diff
@@ -1,10 +1,11 @@
 const gameController = new GameController();
+let selectedDifficulty = queryString = location.search.substring(1);
```

Comment:
Is there a reason to have queryString, because I tested locally like suggestion below and  seems to work well.
```suggestion
let selectedDifficulty = location.search.substring(1);
```

Reply:
> Is there a reason to have queryString?. 😕 I tested locally like suggestion below and seems to work well.

Haha. Obviously, that's my mistake
Your suggestion is correct!